### ISSUE TITLE
Error on project sync

### ISSUE THREAD

**Issue by CarolinePublipresse** [OFFLINE-GmbH/oc-boxes-support#44]
**Describe the bug**
Hello, i just bought your module for a new project, and had these errors on terminal while running project:sync 

**Boxes Version**
v2.1.29

**To Reproduce**
1. Add your boxes on your October Project
2. Create a fresh October install without demo data
3. Sync project. Get errors for "themes/demo/partials/" directory does not exist.
4. Create the demo/partial folder
5. relaunch projet:sync

**Screenshots**
![image](https://user-images.githubusercontent.com/84564072/216619200-30b1facc-8aa3-4d94-8c2a-2d9fa81af3b0.png)

**Code**
 I had to edit file /plugins/offline/boxes/updates/add_locked_column_to_boxes.php to pass through.
```
- $table->json('locked')->after('data');
+ $table->json('locked')->nullable()->after('data');
```

**Comment by tobias-kuendig** [comment]
Thank you, this is fixed in v2.1.30!

**Comment by CarolinePublipresse** [comment]
Nice, thank you !
